app/Http/Middleware/RecordHit.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use App\Models\Hit;
+use Closure;
+use Illuminate\Http\Request;
+
+class RecordHit
+{
+    public function handle(Request $request, Closure $next)
+    {
+        return $next($request);
+    }
+
+    public function terminate($request): void
```

Comment:
I wonder if we could eke an entire blog post out of an intro to terminable middleware?